pkg/registry/client.go

```diff
@@ -416,8 +417,8 @@ func (c *Client) Pull(ref string, options ...PullOption) (*PullResult, error) {
 		}
 	}
 
-	fmt.Fprintf(c.out, "Pulled: %s\n", result.Ref)
-	fmt.Fprintf(c.out, "Digest: %s\n", result.Manifest.Digest)
+	fmt.Fprintf(os.Stderr, "Pulled: %s\n", result.Ref)
```

Comment:
This kind of defeats the purpose of the `out` attribute and it does not solve the issue completely as below you can see another write to `c.out` (in this case it is a warning/error).
I would suggest to configure os.Stderr at the [root](https://github.com/helm/helm/blob/ce66412a723e4d89555dc67217607c6579ffcb21/cmd/helm/root.go#L158):

```go
		registry.ClientOptWriter(os.Stderr),
```